pod-operation/src/components/wheel_encoder.rs

```diff
@@ -0,0 +1,72 @@
+use rppal::gpio::{Gpio, InputPin, Level};
+use std::time::Instant;
+
+const PIN_ENCODER_A: u8 = 1;
+const PIN_ENCODER_B: u8 = 2;
+
+pub struct WheelEncoder {
+	counter: f32,
```

Comment:
Question: is there a particular reason the counter value is floating point? An integer value should be sufficient.